src/ext-test/java/org/opentripplanner/ext/fares/impl/DefaultFareServiceTest.java

```diff
@@ -98,4 +102,50 @@ void unknownLeg() {
     var firstBusLeg = itin.firstTransitLeg().get();
     assertEquals(List.of(firstBusLeg), component.legs());
   }
+
+  @Test
+  void multipleFeeds() {
+    var service = new DefaultFareService();
+    service.addFareRules(FareType.regular, List.of(AIRPORT_TO_CITY_CENTER_SET, OTHER_FEED_SET));
+    var itin = newItinerary(Place.forStop(AIRPORT_STOP))
+      .bus(1, T11_00, T11_05, Place.forStop(CITY_CENTER_A_STOP))
+      .walk(10, Place.forStop(OTHER_FEED_STOP))
+      .bus(OTHER_FEED_ROUTE, 2, T11_20, T11_32, Place.forStop(OTHER_FEED_STOP))
+      .build();
+    var result = service
+      .calculateFares(itin)
+      .getComponents(FareType.regular)
+      .stream()
+      .map(r -> r.fareId())
+      .toList();
+
+    assertEquals(
```

Comment:
Can you please add assertions for the expected prices like is done here? https://github.com/opentripplanner/OpenTripPlanner/blob/bf90bcf6b2bd2b811b987b67d386dd0aab9ff605/src/ext-test/java/org/opentripplanner/ext/fares/impl/DefaultFareServiceTest.java#L75-L100

Reply:
Added those